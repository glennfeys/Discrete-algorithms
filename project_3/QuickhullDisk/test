5
1	0	0	1.5
2	0.5	0.5	1
3	-5	10	1
4	-5	1	1
5	8	0	1
