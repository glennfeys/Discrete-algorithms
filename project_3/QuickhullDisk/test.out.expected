(0, 0)
(-5, 10)
(-5, 1)
(8, 0)
